library/src/main/java/com/novoda/downloadmanager/Migration.java

```diff
@@ -3,9 +3,9 @@
 import java.util.Collections;
 import java.util.List;
 
-class Migration {
+public class Migration {
```

Comment:
I was originally going to use `Migration` for both the `complete` and `partial` downloads BUT I found that the `partials` didn't need most of the information that was here, so opted for using a new model.

Reply:
is the concept of Migration still `relevant` in the library?

Reply:
This class will be removed when I do the `complete` downloads.